packages/block/src/header/header.ts

```diff
@@ -161,7 +161,9 @@ export class BlockHeader {
       blobGasUsed: this.common.isActivatedEIP(4844) ? BIGINT_0 : undefined,
       excessBlobGas: this.common.isActivatedEIP(4844) ? BIGINT_0 : undefined,
       parentBeaconBlockRoot: this.common.isActivatedEIP(4788) ? new Uint8Array(32) : undefined,
-      requestsRoot: this.common.isActivatedEIP(7685) ? KECCAK256_RLP : undefined,
+      // TODO: not sure what the default should be here becuase it would depends on activated EIPs
+      // as even empty requests will produce data to sha hash
+      requestsHash: this.common.isActivatedEIP(7685) ? KECCAK256_RLP : undefined,
```

Comment:
Default hash should be sha256 of the empty string. In that case, if 7685 is activated, but none of the EIPs which actually generate a request root, then this is still correct. 

So it should be `sha256(new Uint8Array())`

Reply:
Added a constant and updated.